VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -102,13 +102,13 @@ public void notifyNetworkConnect(){
 
     @Override
     protected void onCreate(Bundle arg0) {
-        super.onCreate(arg0);
-
         // for landscape player use full screen theme, otherwise, only hide title bar
         if (isLandscape()) {
             setTheme(android.R.style.Theme_NoTitleBar_Fullscreen);
         }
 
+        super.onCreate(arg0);
```

Comment:
Instead of trying to work around the activity lifecycle by moving this super call around, how about overriding the theme via XML in a landscape-only resource?